src/shared/i18n/I18NFrench.cpp

```diff
@@ -18,441 +18,441 @@
+    { I18N_MAINMENU_ENTER_HOSTNAME, "Entrez ici l'adresse IP et le port:" }, 
+    { I18N_MAINMENU_ADDRESS, "Adresse:" },
+    { I18N_MAINMENU_PORT, "Port:" },
+    { I18N_MAINMENU_PASSWORD, "Mot de passe:" },
+    { I18N_MAINMENU_ENTER_PASSWORD, "Entrer le mot de passe:" },
+    { I18N_MAINMENU_SERVER_NAME2, "Nom du serveur:" },
+    { I18N_MAINMENU_GAME_MODE, "Mode de jeu:" },
+    { I18N_MAINMENU_GM_DM, "Match à mort" },
+    { I18N_MAINMENU_GM_TDM, "Match à mort par équipe" },
+    { I18N_MAINMENU_GM_CTF, "Capture de drapeau" },
+    { I18N_MAINMENU_GM_SR, "Course" },
+    { I18N_MAINMENU_GM_CTC, "Possession" },
+    { I18N_MAINMENU_GM_GOH, "Chèvre sur la colline" },
+    { I18N_MAINMENU_MAX_PLAYERS, "max. joueurs:" },
+    { I18N_MAINMENU_WARMUP, "Échauffement:" },
+    { I18N_MAINMENU_DURATION, "Durée:" },
+    { I18N_MAINMENU_IN_SECONDS, "(en secondes)" },
+    { I18N_MAINMENU_IN_MINUTES, "(en minutes)" },
+    { I18N_MAINMENU_SELECT_MAP, "Choisir la carte:" },
+    { I18N_MAINMENU_INVALID_SERVERNAME, "Merci d'entrer un nom de serveur valide" },
+    { I18N_MAINMENU_MAX_PLAYERS_TITLE, "Max. Joueurs" },
+    { I18N_MAINMENU_INVALID_GENERIC_NUMBER, "Merci d'entrer un nombre valide. (${1})" },
+    { I18N_MAINMENU_DURATION2, "Durée" },
+    { I18N_MAINMENU_WARMUP2, "Échauffement" },
+    { I18N_MAINMENU_SELECT_MAP2, "Sélectionner la carte" },
+    { I18N_MAINMENU_SELECT_MAP3, "Merci de sélectionner une carte" },
+    { I18N_MAINMENU_PACKAGE, "Paquet" },
+    { I18N_MAINMENU_HASH, "Hachage" },
+    { I18N_MAINMENU_CREDITS_CODE, "programme:" },
+    { I18N_MAINMENU_CREDITS_GFX, "graphismes:" },
+    { I18N_MAINMENU_CREDITS_MAPS, "cartes:" },
+    { I18N_MAINMENU_CREDITS_MUSIC, "musiques:" },
+    { I18N_MAINMENU_CREDITS_THANKS, "un grand merci à:" },
+    { I18N_MAINMENU_FREDERIC, "et bien sur, frederic la chèvre" },
 
-    { I18N_OPTIONS_PLAYER, "Player" },
-    { I18N_OPTIONS_GRAPHICS_AND_SOUND, "Graphics And Sound" },
-    { I18N_OPTIONS_CONTROLLER, "Controller And Keyboard" },
-    { I18N_OPTIONS_SPECTATE, "Spectate" },
-    { I18N_OPTIONS_SKIP_SONG, "Skip Song" },
-    { I18N_OPTIONS_RETURN, "Return To Main Menu" },
-    { I18N_OPTIONS_SETTINGS11, "player's name:" },
-    { I18N_OPTIONS_SETTINGS12, "show player's name" },
-    { I18N_OPTIONS_SETTINGS13, "character:" },
-    { I18N_OPTIONS_SETTINGS21, "fullscreen graphics mode" },
+    { I18N_OPTIONS_PLAYER, "Joueur" },
+    { I18N_OPTIONS_GRAPHICS_AND_SOUND, "Graphismes et Son" },
+    { I18N_OPTIONS_CONTROLLER, "Manette et Clavier" },
+    { I18N_OPTIONS_SPECTATE, "Regarder" },
+    { I18N_OPTIONS_SKIP_SONG, "Passer la musique" },
+    { I18N_OPTIONS_RETURN, "Retour au menu principal" },
+    { I18N_OPTIONS_SETTINGS11, "nom du joueur:" },
+    { I18N_OPTIONS_SETTINGS12, "montrer le nom du joueur" },
+    { I18N_OPTIONS_SETTINGS13, "personnage:" },
+    { I18N_OPTIONS_SETTINGS21, "plein écran" },
     { I18N_OPTIONS_SETTINGS22, "draw scanlines" },
```

Comment:
Maybe "simuler les lignes de balayage" or just "lignes de balayage" if it's a checkbox.